static/js/services/videowaiter.js

```diff
@@ -46,13 +47,17 @@ define(["underscore"], function(_) {
 				if (videoTracks.length > 0 && this.count >= 10) {
 					var videoTrack = videoTracks[0];
 					if (videoTrack.enabled === true && videoTrack.muted === true) {
+						videoTrack.onunmute = function() {
+							videoTrack.onunmute = undefined;
+							recheck();
```

Comment:
> What's the problem with running cb a second time? Every consumer of the video waiter seems to have no problem with that.

The scope.$emit("active", currentcall) should not be called multiple times even if it seems to work - this single emit is actually the only real reason why the video waiter even exists.